### PR TITLE
Build improvment

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
The remain nxstyle error is:
```
 error: Mixed case identifier found
```
depend on https://github.com/apache/incubator-nuttx/pull/1088

### CODE REVIEW

File: examples/audio_rttl/Makefile [hunk 1]
```diff
@@ -33,10 +33,8 @@
 #
 ############################################################################
 
--include $(TOPDIR)/.config
--include $(TOPDIR)/Make.defs
+include $(TOPDIR)/Make.defs
 -include $(SDKDIR)/Make.defs
-include $(APPDIR)/Make.defs
```
Comment (Ouss4) [inline examples/audio_rttl/Makefile:39]: This makefile needs DELIM, it gets it from apps/Make.defs.  I'm not sure if there is any indirect definition somehow.

File: examples/elf/tests/hello/Makefile [hunk 2]
```diff
@@ -33,16 +33,7 @@
 #
 ############################################################################
 
--include $(TOPDIR)/Make.defs
-
-OBJEXT ?= .o
-DELIM ?= /
-
-ifeq ($(CONFIG_CYGWIN_WINTOOL),y)
-NUTTXLIB = "${shell cygpath -w $(TOPDIR)$(DELIM)staging}"
```
Comment (Ouss4) [inline examples/elf/tests/hello/Makefile:42]: If NUTTXLIB is removed, where doesn't come from? It's used below.

File: examples/audio_rttl/Makefile (hunk 1 above)
Comment (xiaoxiang781216) [reply examples/audio_rttl/Makefile:39]: $(TOPDIR)/Make.defs->$(TOPDIR)/tools/Config.mk->DELIM define here.

Comment (xiaoxiang781216) [reply examples/audio_rttl/Makefile:39]: BTW, there is only one DELIM definition now.

**Pull request merged**